rise/static/rise.yaml

```diff
@@ -0,0 +1,216 @@
+#
+# NOTE that the defaults defined here are essentially NOT used
+# in the code, but if only for documentation purposes it is important
+# that they reflect the defaults that are hardwired in main.js
+#
+Type: Jupyter Notebook Extension
+Compatibility: jupyter-4.3, jupyter-notebook-5.2
```

Comment:
Is this compatibility strict? I mean, if I have notebook 5.4, it will work without issues?

Reply:
Now that I see this again, it's just what was in the file that I copied to start with, I have not tried to find the right values.
Honestly I don't know exactly what the exact impact is here. From a quick test I just made, it looks like this is purely informative; it tells users what they need to have installed to run the extension. 
 It is also possible to remove that line completely, but my feeling is that we should take this chance to spell out what the accurate dependencies are; it's just that I do not know this information for sure.

Reply:
> but my feeling is that we should take this chance to spell out what the accurate dependencies are; it's just that I do not know this information for sure.

You can see that we have specific RISE versions for specific notebook versions: https://damianavila.github.io/RISE/changelog.html#versions
And in setup.py we have notebook >= 5.0.0. We can keep that until a major version of the notebook is released (6.0) and then adjust properly.

Reply:
IMO the information in one `rise.yaml` should depict the actual requirement for **that** RISE version. The fact that other versions are also available is of interest too, but of second order.

I mean, if I open the configurator it means I already have a version of RISE installed and what I need to know is the compatibility versions for that one.

Also, I thought some users had recently reported issues with using 5.0, did I get that wrong ?